sklearn/metrics/_classification.py

```diff
@@ -1776,6 +1978,126 @@ def recall_score(y_true, y_pred, *, labels=None, pos_label=1, average='binary',
     return r
 
 
+def specificity_score(y_true, y_pred, *, labels=None, pos_label=1,
+                      average='binary', sample_weight=None,
+                      zero_division="warn"):
+    """Compute specificity, also known as true negative rate.
+
+    The specificity is the ratio ``tn / (tn + fp)`` where ``tn`` is the number
```

Comment:
You as well single backticks instead of double backticks here and after.

Reply:
Done.